app/services/allocations/finder.rb

```diff
@@ -73,6 +73,8 @@ def apply_date_range_filters(scope)
     end
 
     def split_params(name)
+      return if filter_params[name].blank?
```

Comment:
how about `filter_params[name].presence&.split(',')`

Reply:
I think I prefer the code as @cwrw has it - it's easier to read and understand quickly